### PR TITLE
Fix camera streaming

### PR DESCRIPTION
**Pull request by NiklasNeugebauer**
- set streaming to True by default
- use a should_stream field for persistence (currently if the stream task ends or is not yet started, steaming=False could be written into persistence)
- show the "no image" placeholder if the camera is disconnected

### CODE REVIEW

**Review by falkoschindler — APPROVED** [review]
Looks correct. 👍🏻

File: rosys/vision/camera/camera.py [hunk 1]
```diff
@@ -74,7 +77,7 @@ def get_image_url(self, image: Image) -> str:
 
     def get_latest_image_url(self) -> str:
         image = self.latest_captured_image
-        if image is None:
+        if image is None or not self.is_connected:
```
Comment (falkoschindler) [inline rosys/vision/camera/camera.py:80]: How is this change related to this PR?

Comment (NiklasNeugebauer) [reply rosys/vision/camera/camera.py:80]: it is not
